casbin/internal_enforcer.py

```diff
@@ -88,6 +88,31 @@ def _update_policies(self, sec, ptype, old_rules, new_rules):
 
         return rules_updated
 
+    def _update_filtered_policies(
+        self, sec, ptype, new_rules, field_index, *field_values
```

Comment:
one-line arg list

Reply:
lint test will fail

Reply:
@Shivansh-yadav13 then change the linter config

Reply:
@hsluoyz sorry don't have much experience with this, how can I put the black linter config file in github workspace?

Reply:
I'm not familiar with that too. Do you know that? @leeqvip @ffyuanda

Reply:
@hsluoyz I'm trying to find a solution for this, but rn do you prefer bringing this change to a different PR? which we can create after finding a correct implementation of this till then we can maybe keep an issue open for others to also have a look.

Reply:
@hsluoyz I have fixed it, and now it can detect the config file and use it.